web/src/components/netflow-record/record-field.tsx

```diff
@@ -68,8 +73,9 @@ export const RecordField: React.FC<{
   };
 
   const explicitKubeObjContent = (ip: string, port: number, kind?: string, namespace?: string, name?: string) => {
-    if (name && namespace && kind) {
-      return doubleContainer(namespaceContent(namespace), kubeObjContent(name, kind, namespace), false);
+    // Note: namespace is not mandatory here (e.g. Node objects)
```

Comment:
Good catch ! I will update the `getK8SKindOptions` later if you don't.

Should we get kinds from Loki also ?

Reply:
I wanted to update `getK8SKindOptions` but I'm leaving it for later because it requires more change (the autocompletion workflow changes as it doesn't require a namespace anymore).
I also started to get kinds for Loki, but rolled back, I'm not sure it's worth it in that case, because the list of kinds is more predictable and we should anyway allow free typing if a kind isn't in the list